src/Wyam/packages.config

```diff
@@ -14,6 +14,7 @@
   <package id="Microsoft.Owin.Host.HttpListener" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Owin.Hosting" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Owin.StaticFiles" version="3.0.1" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="8.0.2" targetFramework="net46" />
```

Comment:
Any specific reason JSON.NET was included as a package for the CLI project? Since it's referencing the new module library, it'll bring in the reference for JSON.NET transitivly - this can probably be removed (I can get it if you're not too comfortable working with NuGet).